.env-example

```diff
@@ -32,6 +32,16 @@ OTTM_JIRA_TOKEN=
 # Issue type filters for jira issues, comma separated
 OTTM_JIRA_ISSUE_TYPE=
 OTTM_TARGET_DATABASE=sqlite:///data/${OTTM_SOURCE_PROJECT}.sqlite3
+# MariaDB Authentification
+MARIADB_ROOT_PASSWORD=diouxx
```

Comment:
MARIADB env variables have to be removed, they were just useful for glpi test instance